setup.py

```diff
@@ -585,7 +617,8 @@ def setup_args():
         'build': build,
         'build_ext': build_ext,
         'install_html': install_html,
-        'install': install}
+        'install': install,
+        'check_tests_errors': CheckTestsErrors}
```

Comment:
I see all the other command classes are using lower case.  How about we keep this one lower case as well, to match that convention?  `check_tests_errors`.